src/cmake/thirdparty/SetupGeosxThirdParty.cmake

```diff
@@ -755,8 +755,16 @@ endif()
 ################################
 # Python
 ################################
+message(" CMAKE_VERSION ${CMAKE_VERSION} ")
+if( ${CMAKE_VERSION} VERSION_LESS "3.19" )
+    set( PYTHON_AND_VERSION Python3 )
+    set( PYTHON_OPTIONAL_COMPONENTS)
+else()
```

Comment:
@rrsettgast do you think the `cmake` upgrade will happen soon or it still makes sense to merge this first?